extensions/android-livedata/src/test/java/com/squareup/sqldelight/android/livedata/LiveDataQueryTest.kt

```diff
@@ -0,0 +1,112 @@
+package com.squareup.sqldelight.android.livedata
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+import com.google.common.truth.Truth.assertThat
+import com.squareup.sqldelight.Query
+import com.squareup.sqldelight.android.AndroidSqliteDriver
+import com.squareup.sqldelight.db.SqlDriver
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.Executor
+
+@RunWith(RobolectricTestRunner::class)
+class LiveDataQueryTest {
+  private lateinit var driver: SqlDriver
+
+  @Before
+  fun before() {
+    driver = AndroidSqliteDriver(object : SqlDriver.Schema {
```

Comment:
I feel like we should move the TestDb used by the other extensions into a test-only module that can be shared so everyone isn't responsible for creating a DB and schema.

Reply:
I'm happy to do it as a separate thing so this PR doesn't get out of scope.

I can create an issue and work on it if you are ok with it.

Reply:
I agree separate is best. And it'd be great if you want to contribute it!